### PR TITLE
[type: feat] Support PostgreSQL && update Readme in English

### PR DESCRIPTION
**Pull request by aFlyBird0**
## Description
* Support PostgreSQL
* Update Readme in English
* Fix websocket error( the namespace in websocket url is hardcoded to `shenyu` in the past)

## Note

Now, there is something wrong with the official image `2.5.0` version about pg. You could use my dockerhub image to test.(executed by [`make build-all-image`](https://github.com/apache/shenyu/blob/master/Makefile), based on 115e33accc, yunlongn <yunlongn@outlook.com>, Thu Sep 29 17:09:49 2022 +0800)

```shell
helm install shenyu0930 bird-shenyu/shenyu -n=shenyu --create-namespace \
      --set version=latest \
      --set admin.image=aflybird0/shenyu-admin \
      --set bootstrap.image=aflybird0/shenyu-bootstrap \
      --set dataSource.active=pg \
      --set dataSource.pg.ip=127.0.0.1 \
      --set dataSource.pg.port=5432 \
      --set dataSource.pg.username=postgres \
      --set dataSource.pg.password=123456
```

## Tested in Ubuntu 20.04

image

image

image

image

image

### CODE REVIEW

File: charts/shenyu/README.md [hunk 1]
```diff
@@ -6,14 +6,151 @@ Helm deployment documentation written for Apache/Shenyu.
...
+```shell
+helm install shenyu shenyu/shenyu -n=shenyu --create-namespace \
+      --set dataSource.active=mysql \
+      --set dataSource.mysql.ip=127.0.0.1 \
+      --set dataSource.mysql.port=3306 \
+      --set dataSource.mysql.username=root \
+      --set dataSource.mysql.password=123456 
+```
+
+## PostgreSQL as database(Version of ShenYu > 2.5.0)
```
Comment (erdengk) [inline charts/shenyu/README.md:49]: may be `>=2.5.0` is better? what do you think of this?

File: charts/shenyu/values.yaml [hunk 2]
```diff
@@ -37,6 +37,22 @@ dataSource:
...
     connectorVersion: 8.0.23
+  pg:
+    ip:
+    port: 5432
+    username: postgres
+    password:
+    # pg driver class name
+    driverClass: org.postgresql.Driver
+    connectorVersion: 42.5.0
+  oracle:
```
Comment (erdengk) [inline charts/shenyu/values.yaml:49]: I think oracle should add in another pr

File: charts/shenyu/README.md [hunk 3]
```diff
@@ -6,14 +6,151 @@ Helm deployment documentation written for Apache/Shenyu.
...
+| bootstrap.enabled  | bool   | `true`                      | whether to install shenyu-bootstrap                                                                |
+| bootstrap.image    | string | `"apache/shenyu-bootstrap"` | shenyu-bootstrap image                                                                             |
+| bootstrap.nodePort | int    | `31195`                     | shenyu-bootstrap NodePort                                                                          |
+
+### Database configuration
+
+#### General database configuration
+| configuration-item     | type   | default | description                                     |
+|------------------------|--------|---------|-------------------------------------------------|
+| dataSource.active      | string | `"h2"`  | Database to use, supports `h2`, `mysql`         |
```
Comment (erdengk) [inline charts/shenyu/README.md]: Database to use, supports `h2`, `mysql` 

add `PostgreSQL `

File: charts/shenyu/README.md (hunk 1 above)
Comment (aFlyBird0) [reply charts/shenyu/README.md:49]: 2.5.0(now) is not supported. There is something wrong with `2.5.0` of official image

File: charts/shenyu/README.md (hunk 3 above)
Comment (aFlyBird0) [reply charts/shenyu/README.md]: Got it

File: charts/shenyu/values.yaml (hunk 2 above)
Comment (aFlyBird0) [reply charts/shenyu/values.yaml:49]: Maybe just let it be for now? I have the calibration in the subsequent steps.

image

How about just adding some comments like this?

image

Comment (erdengk) [reply charts/shenyu/values.yaml:49]: very good

File: charts/shenyu/README.md (hunk 1 above)
Comment (erdengk) [reply charts/shenyu/README.md:49]: ok

**Comment by erdengk** [comment]
I think this pr can be merged now, what do you think?
@JooKS-me

**Review by JooKS-me — APPROVED** [review]

**Pull request merged**